app/src/components/Navbar.js

```diff
@@ -0,0 +1,144 @@
+import React, { Fragment } from 'react'
+import { Disclosure, Menu, Transition } from '@headlessui/react'
+import { Bars3Icon, XMarkIcon } from '@heroicons/react/24/outline'
+import { PlusIcon } from '@heroicons/react/20/solid'
+
+const navigation = [
+  { name: 'Dashboard', href: '#'},
+  { name: 'Skills', href: '#'},
+  { name: 'Courses', href: '#'},
+  { name: 'Job Roles', href: '/roles'},
+]
+const userNavigation = [
+  { name: 'Your Learning Journeys', href: '#' },
+  { name: 'Settings', href: '#' },
+  { name: 'Sign out', href: '#' },
+]
+function classNames(...classes) {
+  return classes.filter(Boolean).join(' ')
+}
+
+function isCurrent(href) {
+  let current_url = document.location.toString().split("/")
+  let page = "/" + current_url[current_url.length-1];
+  if (page == href) {
+    return true;
+  }
+  return false;
+}
+
+export default function Navbar() {
+  return (
+    <Disclosure as="nav" className="bg-cyan-800">
+      {({ open }) => (
+        <>
+          <div className="mx-auto max-w-7xl px-4 sm:px-6 lg:px-8">
+            <div className="flex h-16 justify-between">
+              <div className="flex">
+                <div className="-ml-2 mr-2 flex items-center md:hidden">
+                  {/* Mobile menu button */}
+                  <Disclosure.Button className="inline-flex items-center justify-center rounded-md p-2 text-gray-300 hover:bg-cyan-700 hover:text-white focus:outline-none focus:ring-2 focus:ring-inset focus:ring-white">
+                    <span className="sr-only">Open main menu</span>
+                    {open ? (
+                      <XMarkIcon className="block h-6 w-6" aria-hidden="true" />
+                    ) : (
+                      <Bars3Icon className="block h-6 w-6" aria-hidden="true" />
+                    )}
+                  </Disclosure.Button>
+                </div>
+                <div className="flex flex-shrink-0 items-center">
+                  <a href="/" className="text-lg font-bold leading-relaxed inline-block mr-4 py-2 whitespace-nowrap uppercase text-white">LJPS</a>
+                </div>
+                <div className="hidden md:ml-6 md:flex md:items-center md:space-x-4">
+                  {navigation.map((item) => (
```

Comment:
Maybe we can pull this and subsequent similar code blocks below out as a separate function.

Reply:
done in commit e1a92a9